src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -915,6 +915,13 @@ static void window_top_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
             y++;
         imgId = SPR_G2_SANDBOX;
         gfx_draw_sprite(dpi, imgId, x, y, 3);
+
+        // Draw an overlay if clearance checks are disabled
+        if (gCheatsDisableClearanceChecks != 0)
```

Comment:
`gCheatsDisableClearanceChecks` is a bool, so this will do:
```suggestion
        if (gCheatsDisableClearanceChecks)
```